tgui/packages/tgui/interfaces/AdminAntagMenu.js

```diff
@@ -0,0 +1,653 @@
+              fill
+              scrollable
+              buttons={
+                <Stack fill>
+                  <Input
+                    placeholder="Искать..."
+                    width="300px"
+                    onInput={(e, value) => setSearchText(value)}
+                  />
+                  <Button icon="sync" onClick={() => act('refresh')}>
+                    Обновить
+                  </Button>
+                </Stack>
+              }
+            >
+              {PickTab(tabIndex)}
+            </Section>
+          </Stack.Item>
+        </Stack>
+      </Window.Content>
+    </Window>
+  );
+};
+
+const AntagList = (properties, context) => {
+  const { act, data } = useBackend(context);
+  const { antagonists } = data;
+  const [searchText, setSearchText] = useLocalState(context, 'searchText', '');
+  const [sortId, _setSortId] = useLocalState(context, 'sortId', 'antag_names');
+  const [sortOrder, _setSortOrder] = useLocalState(context, 'sortOrder', true);
+
+  const keys = Object.keys(antagonists);
+  if (!keys || keys.length === 0) {
+    return <div>Нет антагонистов.</div>;
+  }
+
+  const antagArray = keys.map((key) => ({ key, ...antagonists[key] }));
+
+  return (
+    <Table className="AdminAntagMenu__list">
+      <Table.Row bold>
+        <SortButton id="name">Имя моба</SortButton>
+        <SortButton id="">Кнопки</SortButton>
+        <SortButton id="antag_names">Тип(-ы) антагонистов</SortButton>
+        <SortButton id="status">Статус</SortButton>
+      </Table.Row>
+      {antagArray
+        .filter(
+          createSearch(searchText, ({ key, name, status, antag_names }) => {
+            return name + '|' + status + '|' + antag_names.join(', ');
+          })
+        )
+        .sort((a, b) => {
+          const i = sortOrder ? 1 : -1;
+          if (a[sortId] === undefined || a[sortId] === null) {
+            return i;
+          }
+          if (b[sortId] === undefined || b[sortId] === null) {
+            return -1 * i;
+          }
+          if (typeof a[sortId] === 'number') {
+            return (a[sortId] - b[sortId]) * i;
+          }
+          let firstValue = a[sortId];
+          let secondValue = b[sortId];
+          if (sortId === 'antag_names') {
+            firstValue = a[sortId].join(', ');
+            secondValue = b[sortId].join(', ');
+          }
+          return firstValue.localeCompare(secondValue) * i;
+        })
+        .map(
+          (
+            {
+              key,
+              name,
+              body_destroyed,
+              is_hijacker,
+              antag_mind_uid,
+              ckey,
+              status,
+              antag_names,
+            },
+            index
+          ) => (
+            <Table.Row key={index}>
+              <Table.Cell collapsing>
+                {!body_destroyed ? (
+                  <Button
+                    color={is_hijacker || !name ? 'red' : ''}
+                    tooltip={is_hijacker ? 'Хиджакер' : ''}
```

Comment:
```suggestion
                    tooltip={is_hijacker ? 'Угонщик шаттла' : ''}
```
Харош так сильно англицизмами сыпать, ну.